### DISCUSSION TITLE
FEATURE REQUEST Add Twitch channel points integration

### DISCUSSION THREAD

**Discussion by MorbidMorke** [howroyd/simonsays#40] [Ideas]
Add a channel points integration so we can choose between chat and points - especially helpful for the larger riskier ones like rekt, etc

(I know this is already in consideration, but thought I'd note it down here in discussions)

**Comment by MorbidMorke** [comment]
RE using via Firebot

From dev: 
could add a checkbox in the gui that is "privileged only" so basically has to be on the sudo or bot list for it to work. 

Bot posts as streamer in chat: hmm, the only issue with that, because adding the broadcaster to the list of bots is easy, is that then the broadcaster always ignores cooldowns and random chances.  Which makes testing those two features impossible.  I might just add a "test cooldowns mode" button which then kicks the streamer out of the bot list temporarily.

**Comment by howroyd** [comment]
> Add a channel points integration so we can choose between chat and points - especially helpful for the larger riskier ones like rekt, etc
> 
> (I know this is already in consideration, but thought I'd note it down here in discussions)

Help required for channel point and/or bits integration.  This does not come through the Twitch IRC that we currently connect to.  We'd need another server connection (with different protocol) which also requires authentication.
